### PR TITLE
[SD-3627][SD-3493][SD-2564][SD-3532] Expire the content and move to archived database.

### PR DESCRIPTION
**Pull request by mdhaman**
This PR contains expiring the content and moving the expired content to separate `archived` database and it is dependent on  [#PR108](https://github.com/superdesk/superdesk-core/pull/108).

Steps To be Executed after Merge:
1)  Download the script [clean_data.txt](https://github.com/superdesk/superdesk/files/64790/clean_data.txt) and rename it to clean_data.js.
2) Run the command to purge data from mongo database. `mongo localhost:27017/superdesk clean_data.js`.
3) Run the command to delete elasticsearch index: `curl -XDELETE http://<elasticsearch_url>/superdesk`

Please merge [Company Codes PR] (https://github.com/superdesk/superdesk/pull/1547) and this PR together.

### CODE REVIEW

**Comment by akintolga** [comment]
:+1:

File: server/apps/publish/published_item.py [hunk 1]
```diff
@@ -96,6 +73,18 @@ class PublishedItemResource(Resource):
 
 
 class PublishedItemService(BaseService):
+
+    def raise_if_not_marked_for_publication(self, doc):
```
Comment (sivakuna-aap) [inline server/apps/publish/published_item.py]: This should be after class's doc string

**Comment by sivakuna-aap** [comment]
:+1: except the minor one

**Comment by ioanpocol** [comment]
:+1:

**Pull request merged**